package.json

```diff
@@ -11,8 +11,7 @@
     "single-page-app-codegen": "index.js"
   },
   "scripts": {
-    "test-env": "zenv",
-    "test-integration": "zenv integration",
+    "test-integration": "bash ./test/testenv_cli.sh",
     "test-integration-b": "bash ./test/previous/test/integration-test.sh"
```

Comment:
"test-integration-b" can be removed as well as the "/test/previous" dir.

Reply:
Done.